src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,163 @@
+                            "%s\n" +
+                            "Now you have %d tasks in the list.", newTask, tasklist.size());
+                    echo(taskStatus);
+                    writeToFile(tasklist);
+                }
+
+            } else if (command.contains("deadline")) {
+                String deadlineTask = command.replace("deadline ", "");
+                if (deadlineTask.equals(command) || "".equals(deadlineTask)) {
+                    String error = DukeException.taskErrorMessage(command);
+                    echo(error);
+                } else {
+                    String[] parts = deadlineTask.split(" /by ", 2);
+                    Deadline newTask = new Deadline(parts[0], parts[1]);
+                    tasklist.add(newTask);
+                    String taskStatus = String.format("Got it. I've added this task:\n" +
+                            "%s\n" +
+                            "Now you have %d tasks in the list.", newTask, tasklist.size());
+                    echo(taskStatus);
+                    writeToFile(tasklist);
+                }
+            } else if (command.contains("event")) {
+                String eventTask = command.replace("event ", "");
+                if (eventTask.equals(command) || "".equals(eventTask)) {
+                    String error = DukeException.taskErrorMessage(command);
+                    echo(error);
+                } else {
+                    String[] parts = eventTask.split(" /",2);
+                    Task newTask = new Event(parts[0], parts[1]);
+                    tasklist.add(newTask);
+                    String taskStatus = String.format("Got it. I've added this task:\n" +
+                            "%s\n" +
+                            "Now you have %d tasks in the list.", newTask, tasklist.size());
+                    echo(taskStatus);
+                    writeToFile(tasklist);
+                }
+            } else if (command.contains("delete")) {
+                String deleteTaskNumber = command.replace("delete ", "");
+                if (deleteTaskNumber.equals(command) || "".equals(deleteTaskNumber)) {
+                    String error = DukeException.taskErrorMessage(command);
+                    echo(error);
+                } else {
+                    int n = Integer.parseInt(deleteTaskNumber);
+                    if (n > tasklist.size()){
+                        echo(DukeException.IndexOutofBoundsException(tasklist));
+                    } else {
+                        Task deletedTask = tasklist.remove(n-1);
+                        String taskStatus = String.format("Noted. I've removed this task:\n" +
+                                "%s\n" +
+                                "Now you have %d tasks in the list.", deletedTask, tasklist.size());
+                        echo(taskStatus);
+                        writeToFile(tasklist);
+                    }
+                }
+            } else if(command.contains("find")) {
```

Comment:
Should there be a space here? I noticed the same issue in several other places too.
```suggestion
            } else if (command.contains("find")) {
```